ui/scripts/ui-custom/migrate.js

```diff
@@ -0,0 +1,127 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
@koushik-das I have provided screen shots in JIRA. Attaching them here as well.
![migrateinstance-seehosts-search](https://cloud.githubusercontent.com/assets/13415006/22204992/fdfacbae-e19a-11e6-9c7b-59a0776c68e0.PNG)
![migrateinstance-seehosts](https://cloud.githubusercontent.com/assets/13415006/22204993/fdfaf232-e19a-11e6-8ea9-1af368f1f6d7.PNG)